FogCreek/Glitch-Community
# Comfortable buzzard: fix notification button styles

## Links
* Remix link: http://comfortable-buzzard.glitch.me
* Issue-tracker link: https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/482

## GIF/Screenshots:
![Screen Shot 2019-10-01 at 4 53 15 PM](https://user-images.githubusercontent.com/938722/65999763-0c91e680-e46c-11e9-803b-a80039e8fcb3.png)

## Changes:
* move `<LocalStyle>` up in the component hierarchy, so that it wraps `<NotificationsProvider>`

## How To Test:
* log into http://comfortable-buzzard.glitch.me
* add a project to a collection
* the "Take me there" link should be styled as a button
